checkov/terraform/checks/resource/gcp/GKESecureBootforShieldedNodes.py

```diff
@@ -15,7 +15,7 @@ def scan_resource_conf(self, conf):
             node = conf["node_config"][0]
             if isinstance(node, dict) and 'shielded_instance_config' in node:
                 monitor = node["shielded_instance_config"][0]
-                if monitor["enable_secure_boot"] == [True]:
+                if monitor.get("enable_secure_boot", None) == [True]:
```

Comment:
to set the default value to `None` is not needed, because it is actually the default anyway 😉